py/edgeseleniumtools/test/edge_driver_test.py

```diff
@@ -0,0 +1,102 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+import sys
+from edgeseleniumtools import webdriver
+
+class EdgeDriverTest(unittest.TestCase):
```

Comment:
Please make sure every test in this file (except the *to_capabilities tests) actually creates an Edge() object. The tests should also check the driver.capabilities object instead of the options. The driver.capabilities object contains the response from the WebDriver server after the session is created and can be used to check whether the correct browser version was launched. See UsesLegacy and UsesChromium in the dotnet tests.

Reply:
Please validate the driver.capabilities values after launching the browser.